pkg/app/server/grpcapi/envoy_ext_authz_api.go

```diff
@@ -0,0 +1,68 @@
+package grpcapi
```

Comment:
Please add the pipecd license header (refer other files)

Reply:
Thanks. I added this commit: https://github.com/pipe-cd/pipecd/pull/4990/commits/6ed9c7aaebec4b6141bd37aca43a1742024a86ae.